redbot/core/bot.py

```diff
@@ -134,24 +134,50 @@ def __init__(self, *args, cli_flags=None, bot_dir: Path = Path.cwd(), **kwargs):
 
         self._permissions_hooks: List[commands.CheckPredicate] = []
 
-    async def maybe_update_config(self):
+    async def get_embed_color(self, location: discord.abc.Messageable) -> discord.Color:
+        """
+        Get the embed color for a location.
+
+        Parameters
+        ----------
+        location : `discord.abc.Messageable`
+
+        Returns
+        -------
+        discord.Color
+        """
+
+        guild = getattr(location, "guild", None)
+
+        if (
+            guild
+            and await self._config.guild(guild).use_bot_color()
+            and not isinstance(location, discord.Member)
+        ):
+            return guild.me.color
+
+        return self._color
+
+    get_embed_colour = get_embed_color
+
+    async def _maybe_update_config(self):
```

Comment:
This method is currently shown in the docs <https://red-discordbot.readthedocs.io/en/latest/framework_bot.html#redbot.core.bot.RedBase.maybe_update_config>. Should it be added to the breaking changes now that it is private?

Reply:
I dont think so. Just because it was documented doesnt mean it was ever intended for use externally (part of why some of these changes are being made)

No 3rd party code could ever run this function safely before.